test

```diff
@@ -58,6 +58,7 @@ function unit_tests {
 
 function integration_tests {
 	echo "Running integration tests..."
+	MIGRATION_FROM=v3.0.2
```

Comment:
This might be better with environmental variable. I put this just for this PR to test it in the CI.